docs/sources/http-routingtable.md

```diff
@@ -8,8 +8,8 @@
 [**/version**](#cap-/version)
 
   -- -------------------------------------------------------------------------------------------------------------------------------------------------------------------- ----
-                                                                                                                                                                          
-     **/api**                                                                                                                                                             
+      
+     **/api**
```

Comment:
Not really sure where this document is visible on the site, but these could be intended as well